src/client/components/pages/parts/delete-or-remove-collaboration-modal.js

```diff
@@ -0,0 +1,128 @@
+import * as bootstrap from 'react-bootstrap';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import PropTypes from 'prop-types';
+import React from 'react';
+import request from 'superagent';
+
+
+const {Alert, Button, Modal} = bootstrap;
+
+class DeleteOrRemoveCollaborationModal extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			error: null
+		};
+
+		this.handleSubmit = this.handleSubmit.bind(this);
+	}
+
+	handleSubmit() {
+		request.post(this.postUrl)
+			.send(this.postData)
+			.then((res) => {
+				if (this.props.isDelete) {
+					window.location.href = `/editor/${this.props.collection.ownerId}/collections`;
+				}
```

Comment:
I wonder if this is necessary… Considering you would need to be the collection's owner to delete it, props.userId should be the same as props.collection.ownerId.